### PR TITLE
Fix format pubDate in tutorials

### PR DESCRIPTION
**Pull request by AitorMT**
#### Description (required)

Fixed date formatting in tutorials  #6718

### CODE REVIEW

**Comment by Yan-Thomas** [comment]
Thanks @AitorMT, the change makes sense. Since we have the tutorial in more languages, can you change only the English docs here so translators can individually update this in their languages? 🙌

**Comment by AitorMT** [comment]
> Thanks @AitorMT, the change makes sense. Since we have the tutorial in more languages, can you change only the English docs here so translators can individually update this in their languages? 🙌

I just updated it so that it is only in the `en` version, I don't know if this is the right thing to do, but I got confused. 😅😅

File: src/content/docs/es/tutorial/4-layouts/2.mdx [hunk 1]
```diff
@@ -95,7 +95,7 @@ Este es un ejemplo de un plantilla refactorizada que deja sólo el contenido ind
 const { frontmatter } = Astro.props;
 ---
 <h1> {frontmatter.title}</h1>
-<p>{frontmatter.pubDate.slice(0,10)}</p>
+<p>{frontmatter.pubDate.toString().slice(0,10)}</p>
```
Comment (AitorMT) [inline src/content/docs/es/tutorial/4-layouts/2.mdx:98]:
```suggestion
<p>{frontmatter.pubDate.slice(0,10)}</p>
```

File: src/content/docs/es/tutorial/4-layouts/3.mdx [hunk 2]
```diff
@@ -58,7 +58,7 @@ El `MarkdownPostLayout.astro` te proporciona algunas plantillas adicionales para
     ```astro title="src/layouts/MarkdownPostLayout.astro" del={2}
     <BaseLayout pageTitle={frontmatter.title}>
       <h1>{frontmatter.title}</h1>
-      <p>{frontmatter.pubDate.slice(0,10)}</p>
+      <p>{frontmatter.pubDate.toString().slice(0,10)}</p>
```
Comment (AitorMT) [inline src/content/docs/es/tutorial/4-layouts/3.mdx:61]:
```suggestion
      <p>{frontmatter.pubDate.slice(0,10)}</p>
```

File: src/content/docs/es/tutorial/5-astro-api/3.mdx [hunk 3]
```diff
@@ -345,7 +345,7 @@ const { frontmatter } = Astro.props;
 --- 
 <BaseLayout pageTitle={frontmatter.title}>
   <p><em>{frontmatter.description}</em></p>
-  <p>{frontmatter.pubDate.slice(0,10)}</p>
+  <p>{frontmatter.pubDate.toString().slice(0,10)}</p>
```
Comment (AitorMT) [inline src/content/docs/es/tutorial/5-astro-api/3.mdx:348]:
```suggestion
  <p>{frontmatter.pubDate.slice(0,10)}</p>
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**